src/bindings/DatePicker.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import DateRangePicker from '@wojtekmaj/react-daterange-picker';
+
+export default class DatePicker extends Component {
+  state = {
+    date: [new Date(), new Date()],
+  }
+
+  onChange = date => this.setState({ date })
```

Comment:
you can write this in Reason with hooks 🤔

Reply:
yeah, I was just basing myself to validate the POC